src/exasol_jdbc/connectionProperties.js

```diff
@@ -42,12 +42,11 @@
         const hostName = attr[connectionHelper.attributeServer];
         props["kerberoshostname"] = hostName;
         props["kerberosservicename"] = "exasol";
-    }
-
-    if (runningOnServer()) {
-        props["user"] = getServerUser();
-        props["loginType"] = "2";
-        props["logintype"] = "gss";
+        if (runningOnServer()) {
+            props["user"] = getServerUser();
+            props["loginType"] = "2";
+            props["logintype"] = "gss";
```

Comment:
"t" vs "T" -- Seriously? 😆

Reply:
This was changed in the JDBC driver, and we need to use both variants to be compatible with new and old drivers :)